convex/game/players.ts

```diff
@@ -58,7 +58,15 @@ export const players = defineTable({
 
   // Pointer to the locations table for the player's current position.
   locationId: v.id('locations'),
-}).index('active', ['worldId', 'active', 'human']);
+};
+
+export const playerDoc = v.object({
+  _id: v.id('players'),
+  _creationTime: v.number(),
+  ...playerValidator,
+});
```

Comment:
I'm curious what you thought of the old `Table` util:
https://github.com/get-convex/ai-town/blob/bb26c28aadc09515bdb6639a39ead059fe7b7d43/convex/lib/utils.ts#L78
```
const Players = Table('players', { ...fields });
Players.doc
Players.fields
defineSchema({players: Players.table.index('active', ...
```